### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by pkriens**
Extensive modernizing of the readme ...

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,88 +1,112 @@
...
-[CodeQL](https://github.com/bndtools/bnd/security/code-scanning?query=tool%3ACodeQL) is used for continuous security analysis.
+* `master` – Where the work for the next release takes place. Pushes on this branch release to the snapshot repository
+* `classic` – A continuation of 6.4 compiled on Java 1.8 for the poor developers stuck in the limbo of 1.8 and 17.
+* `next` – Release branch to Maven central. Is used during the [release process](https://github.com/bndtools/bnd/wiki/Release-Process).
 
-A Maven snapshot repository of the latest build is available at <https://bndtools.jfrog.io/bndtools/libs-snapshot-local>.
+We generally do not use other branches. Issues and features are handled through PR's from forked repos.
 
-### Using the latest development SNAPSHOT build of Bnd/Bndtools
+### Issues
```
Comment (bjhargrave) [inline README.md:53]: Removing the 2 "Using the latest development" sections is a mistake. They are necessary to refer people to when asking them to test some change in master or to try an RC build. I think these sections are linked to from other places.

File: README.md [hunk 2]
```diff
@@ -1,88 +1,112 @@
...
+### Milestones
 
-![Powered by Artifactory](https://github.com/bndtools/bnd/raw/master/docs/img/Powered-by-artifactory_04.png)
+We try to have a release every 3-6 months and plan these with [milestones](https://github.com/bndtools/bnd/milestones).
 
-## Release
+### PRs
 
-Release versions of Bnd artifacts including Maven and Gradle plugins are available from [Maven Central](https://search.maven.org/search?q=g:biz.aQute.bnd). Release versions of the Bnd Gradle plugins are also available from the [Gradle Plugin repository](https://plugins.gradle.org/search?term=biz.aQute.bnd).
+[PRs](https://github.com/bndtools/bnd/pulls) should be submitted via another repo. We are using the following branch names:
```
Comment (bjhargrave) [inline README.md:65]: I don't think there is much use in attempting to prescribe the branch names people use in their own forks. The main thing is that they don't make a PR from master. You may also want to make clear that all PRs must be against master. Changes to classic are a special case and perhaps left to the maintainers alone.

File: README.md [hunk 3]
```diff
@@ -1,88 +1,112 @@
...
-- `./mvnw -Pdist deploy` - Assembles and publishes the Bnd Maven plugins into `dist/bundles`.
+### Branches 
 
-Bnd/Bndtools is continuously built on [GitHub Actions](https://github.com/bndtools/bnd/actions/workflows/cibuild.yml).
-[![GitHub Actions CI Build Status](https://github.com/bndtools/bnd/actions/workflows/cibuild.yml/badge.svg)](https://github.com/bndtools/bnd/actions/workflows/cibuild.yml)
+We have the following [branches](https://github.com/bndtools/bnd/branches/all):
 
-[CodeQL](https://github.com/bndtools/bnd/security/code-scanning?query=tool%3ACodeQL) is used for continuous security analysis.
+* `master` – Where the work for the next release takes place. Pushes on this branch release to the snapshot repository
+* `classic` – A continuation of 6.4 compiled on Java 1.8 for the poor developers stuck in the limbo of 1.8 and 17.
```
Comment (bjhargrave) [inline README.md:48]: How are you maintaining the dependencies on the classic branch? Do you keep them all updated like master? What version of bnd is used to build classic? The latest bnd snapshot from master? There should be some description of how classic is kept up to date.

Comment (pkriens) [reply README.md:48]: I've added a section to the wiki and will update it after the first release of classic.

**Comment by pkriens** [comment]
@bndtools/contributors Any more feedback? If not I will update the readme page end of today

**Pull request merged**